accord-core/src/main/java/accord/local/SerializerSupport.java

```diff
@@ -162,15 +167,15 @@ private static Command.Truncated truncated(Mutable attrs, SaveStatus status, Tim
             case TruncatedApplyWithDeps:
                 Set<MessageType> witnessed = messageProvider.test(APPLY_TYPES);
                 checkState(!witnessed.isEmpty());
-                if (witnessed.contains(APPLY_MINIMAL_REQ))
+                if (witnessed.contains(APPLY_MAXIMAL_REQ))
```

Comment:
there was a bug where the `partialTxn` exists in `APPLY_MINIMAL_REQ` but because the next section is an `if` and not an `else if`  the `PROPAGATE_APPLY_MSG` was seen, but that doesn't have the `partialTxn`; this lead to a NPE...

I changed to use else for each section and go from max -> min -> propagate